mmv1/products/metastore/api.yaml

```diff
@@ -0,0 +1,219 @@
+          The default value is `global`.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'name'
+        output: true
+        description: |
+          The relative resource name of the metastore service.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: 'labels'
+        description: 'User-defined labels for the metastore service.'
+      # This is an x-product resource reference.
+      - !ruby/object:Api::Type::String
+        name: 'network'
+        input: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. It is specified in the following form:
+
+          "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: 'endpointUri'
+        output: true
+        description: |
+          The URI of the endpoint used to access the metastore service.
+      - !ruby/object:Api::Type::Integer
+        name: 'port'
+        description: |
+          The TCP port at which the metastore service is reached. Default: 9083.
+      - !ruby/object:Api::Type::String
+        name: 'state'
+        output: true
+        description: |
+          The current state of the metastore service.
+      - !ruby/object:Api::Type::String
+        name: 'stateMessage'
+        output: true
+        description: |
+          Additional information about the current state of the metastore service, if available.
+      - !ruby/object:Api::Type::String
+        name: 'artifactGcsUri'
+        output: true
+        description: |
+          A Cloud Storage URI (starting with gs://) that specifies where artifacts related to the metastore service are stored.
+      - !ruby/object:Api::Type::Enum
+        name: 'tier'
+        description: |
+          The tier of the service.
+        values:
+          - :DEVELOPER
+          - :ENTERPRISE
+      - !ruby/object:Api::Type::NestedObject
+        name: 'maintenanceWindow'
+        description: |
+          The one hour maintenance window of the metastore service.
+          This specifies when the service can be restarted for maintenance purposes in UTC time.
+        properties:
+          - !ruby/object:Api::Type::Integer
+            name: 'hourOfDay'
+            description: |
+              The hour of day (0-23) when the window starts.
+          - !ruby/object:Api::Type::Enum
+              name: 'dayOfWeek'
+              description: |
+                The day of week, when the window starts.
+              default_value: :DAY_OF_WEEK_UNSPECIFIED
+              values:
+              - :DAY_OF_WEEK_UNSPECIFIED
+              - :MONDAY
+              - :TUESDAY
+              - :WEDNESDAY
+              - :THURSDAY
+              - :FRIDAY
+              - :SATURDAY
+              - :SUNDAY
+      - !ruby/object:Api::Type::NestedObject
+        name: 'metadataManagementActivity'
+        output: true
```

Comment:
Now that this whole field is output-only, is there a reason to include it? Why would a user want this value to show up in their Terraform resource?

Reply:
state, start and end probably are not useful. Good point as they are transient fields and their values in state wouldn't be very helpful/relevant. For the other field, `destinationGcsUri` I could see being useful. But what do we consider useful from our perspective as terraform developers. True its maybe not configurable but argument could be made that it is useful to have it visible in the state. I guess the counter argument could be made that the user can look up the state within gcs?

Reply:
I've removed it for now :)